queries.sql

```diff
@@ -6,3 +6,40 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species='';
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
```

Comment:
Please note that you should have a transaction where you:
- DELETE all animals born after Jan 1st, 2022
Create a savepoint for the transaction

- UPDATE all animals' weight to be their weight multiplied by -1.
Rollback to the savepoint

- UPDATE all animals' weights that are negative to be their weight multiplied by -1.
Commit transaction